### DISCUSSION TITLE
how to install vcpkg packages with Cmake?

### DISCUSSION THREAD

**Discussion by DMC1290** [microsoft/vcpkg#40744] [Q&A]
I'm on microsoft's website trying to install VCPKG with CMake and the second step is already a problem for me i just don't know where i should whrite is i don't even know how to find the "console".
If someone has enough time and kindness to help me a little bit i would be thankful
(i'm using windows 11)

**Comment by autoantwort** [comment]
To open the console: Press windows key -> Search for *Terminal* -> Press enter. Then you have a console.

**Comment by DMC1290** [comment]
Oh thanks
